_data/site.yaml

```diff
@@ -5,7 +5,7 @@ description: >- # this means to ignore newlines
   Aenean et sapien a leo auctor scelerisque quis nec magna. Sed dictum ante a risus vehicula facilisis.
 
 # Twitter handle. Only the handle, not the URL.
-twitter: 18F
+# twitter: 18F
```

Comment:
Would this make sense to put a "variable" there?

```suggestion
# twitter: <your-agency-handle>
```

Reply:
we don't use that convention elsewhere in the file. It doesn't seem likely to confuse people to me.

Reply:
Fair. Should we change it to `pages` or `cloud-gov` to stay on branc?

Reply:
replaced with commented yaml
```# twitter: agency_name```